### PR TITLE
LG-3535: Palette click-to-copy

### PR DESCRIPTION
**Pull request by spark33**
## ✍️ Proposed changes

🎟 _Jira ticket:_ [LG-3535](https://jira.mongodb.org/browse/LG-3535)

### For bug fixes, new features & breaking changes

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [x] I have run `yarn changeset` and documented my changes

### For new components

- [ ] I have added my new package to the global tsconfig
- [ ] I have added my new package to the Table of Contents on the global README
- [ ] I have verified the Live Example looks as intended on the design website.

### CODE REVIEW

**Comment by bruugey** [comment]
Looks like we need to `rm` tokens from palette or add it as a dep. 
`@leafygreen-ui/palette is missing devDependencies: @leafygreen-ui/tokens`

File: packages/palette/src/Palette.story.tsx [hunk 1]
```diff
@@ -36,20 +29,37 @@ interface ColorBlockProps extends HTMLElementProps<'div'> {
...
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-family: 'Euclid Circular A', 'Helvetica Neue', Helvetica, Arial, sans-serif;
+  border-radius: inherit;
+  z-index: 9999;
```
Comment (shaneeza) [inline packages/palette/src/Palette.story.tsx:43]: why so high?

**Review by shaneeza — CHANGES_REQUESTED** [review]
There's no focus state on the buttons, we should add that in

File: packages/palette/src/Palette.story.tsx [hunk 2]
```diff
@@ -36,20 +29,41 @@ interface ColorBlockProps extends HTMLElementProps<'div'> {
...
   outline: none;
   border: none;
   border-top-color: transparent;
   width: 100%;
   padding-bottom: 100%;
   border-radius: 8px;
   cursor: pointer;
+  vertical-align: top;
+
+  &:focus {
```
Comment (shaneeza) [inline packages/palette/src/Palette.story.tsx:64]: let's make this focus-visible? so it doesn't show up on mouse click

**Review by shaneeza — APPROVED** [review]

**Pull request merged**